pkg/storage/bucket/client.go

```diff
@@ -84,6 +85,7 @@ func (cfg *Config) RegisterFlagsWithPrefixAndDefaultDirectory(prefix, dir string
 	cfg.Filesystem.RegisterFlagsWithPrefixAndDefaultDirectory(prefix, dir, f)
 
 	f.StringVar(&cfg.Backend, prefix+"backend", Filesystem, fmt.Sprintf("Backend storage to use. Supported backends are: %s.", strings.Join(cfg.supportedBackends(), ", ")))
+	f.StringVar(&cfg.StoragePrefix, prefix+"storage-prefix", "", fmt.Sprintf("Prefix for backend bucket/container. All files will be stored under this prefix."))
```

Comment:
[nit] I wouldn't mention "container" cause could be misleading to most people.

```suggestion
	f.StringVar(&cfg.StoragePrefix, prefix+"storage-prefix", "", fmt.Sprintf("Prefix for all objects stored in the backend storage."))
```

Reply:
imagine writing docs for microsoft azure...